src/main/java/net/minecraftforge/common/gui/GuiContainerPair.java

```diff
@@ -0,0 +1,18 @@
+package net.minecraftforge.common.gui;
+
+import net.minecraft.client.gui.Gui;
+import net.minecraft.inventory.Container;
+
+
+    public class GuiContainerPair {
+
+        public final Container container;
+        public final Gui gui;
+
+        public GuiContainerPair(Container container, Gui gui){
+            this.container=container;
+            this.gui=gui;
+        }
+
+    }
```

Comment:
So the entire point of this class is to hard tie a client and server classes together therefor killing ANY chance the dedicated server works on it's own?